src/python/pants/backend/python/dependency_inference/default_module_mapping.py

```diff
@@ -216,7 +216,6 @@ def two_groups_hyphens_two_replacements_with_suffix(
     "pyjwt": ("jwt",),
     "pykube-ng": ("pykube",),
     "pymongo": ("bson", "gridfs", "pymongo"),
-    "pymupdf": ("fitz",),
```

Comment:
In order not to break users of older versions of this package, you probably want to add `pymupdf` to the list of provided packages (along with `fitz`), rather than remove this mapping entirely.

Reply:
Wow.. you're right lol.

Reply:
I'm otw to office. Will fix this soon